lib/mint/web_socket/frame.ex

```diff
@@ -21,6 +21,34 @@ defmodule Mint.WebSocket.Frame do
   defrecord :ping, shared
   defrecord :pong, shared
 
+  @type continuation_frame() ::
+          record(:continuation, reserved: <<_::3>>, mask: binary, data: binary, fin?: boolean)
```

Comment:
Let's make a `@typep` for `reserved` to cut down on the repetition here

Also, let's use the `type()` syntax rather than `type` to be consistent with other types/specs in the project